docs/community-ee/community-ee-announcement.md

```diff
@@ -76,17 +64,10 @@ Ansible community-ee-minimal is a container image. It includes:
 ## To know about future releases
 
 Join the [Ansible Community Forum](https://forum.ansible.com) to follow along and participate
-in all the discussions and release related discussions and
-announcements. Feel free to share your thoughts, ideas and concerns
-there.
-
-Subscribe to the Bullhorn for all future release dates, announcements,
-and Ansible community contributor news at:
-
-[https://bit.ly/subscribe-bullhorn](https://bit.ly/subscribe-bullhorn)
+in all the discussions and release related discussions and announcements. Feel free to share your
+thoughts, ideas and concerns there.
```

Comment:
So I think we should reconsider this section. Rather than an open call to join the forum, I'd suggest just trimming this down to ''...follow along and participate in the discussions". And leave it at that. You can even consider just pointing them to the ecosystem forum page instead of the full forum if you want to focus this more.